src/main/java/seedu/address/model/meeting/Meeting.java

```diff
@@ -11,7 +15,7 @@
 public class Meeting {
 
     public static final String MESSAGE_MEETING_CONSTRAINTS =
-            "Meeting should only contain a date and time in DDMMYYHHMM format";
+            "Meeting should only contain a date and time in DD/MM/YY HHMM format";
```

Comment:
Might want to include "slashes optional" or something

Reply:
Good idea!